CCS2.0/DataLibrary/BusinessLogic/PhotoProccesor.cs

```diff
@@ -144,7 +144,9 @@ public static List<ImageModel> FindImg(string imgId)
             string sql = @"select *
                         from dbo.Image
 
-                        Where Grade Like '%" + @imgId + "%';";
+                        Where Name Like '%" + @imgId + "%' OR Email Like '%" +  @imgId + "%'" +
+                        "OR School_Year_Begin Like '%" + @imgId + "%' OR School_Year_End Like '%" + @imgId + "%'" +
+                        "OR Grade Like '%" + @imgId + "%' OR Teacher_Name Like '%" + @imgId + "%';";
```

Comment:
Excellent. Though can you paramaterize this query? XSS is an issue. Dapper should do that for you.